examples/actix-http-tracing/src/main.rs

```diff
@@ -18,19 +20,23 @@ fn greet_user(username: &str) -> String {
 
 #[actix_web::main]
 async fn main() -> io::Result<()> {
-    // Start an (optional) otel prometheus metrics pipeline
-    let metrics_exporter = opentelemetry_prometheus::exporter().init();
-    let request_metrics = actix_web_opentelemetry::RequestMetrics::new(
-        opentelemetry::global::meter("actix_http_tracing"),
-        Some(|req: &actix_web::dev::ServiceRequest| {
-            req.path() == "/metrics" && req.method() == actix_web::http::Method::GET
-        }),
-        Some(metrics_exporter),
-    );
+    let controller = controllers::basic(
+        processors::factory(
+            selectors::simple::histogram([1.0, 2.0, 5.0, 10.0, 20.0, 50.0]),
```

Comment:
Default histogram should aggregate with more buckets: 
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#explicit-bucket-histogram-aggregation